book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,261 @@
+Conversely, a tool that is a good fit gets out of people's way and makes their jobs easier can be a major boost to productivity and morale.
+
+(rr-rdm-elns-choosing-failure-modes)=
+### Failure modes in user adoption of ELNs
+
+Incomplete or inflexible ELN solutions which increase friction in workflows can cause issues.
+People will avoid using ill-fitting tools that they perceive them to be getting in their way, especially scientists.
+They are creative, stubborn, resourceful, and impatiently focused on anwsering their research questions with watever tools are available to them. 
+Finding software with the flexibility, capability, and scope to keep up is not easy and if it doesn't researchers won't use it - at least not as you intended.
+
+-   Shadow IT/notebooks
+    -   People using their own unsactioned solutions which may be a compliance issue and introduce risks that you have not accounted for
+-   Infrequent updates
+    -   People dump a copy of their work into the ELN only as often as necessary for compliance
+-   Consistency issues
+    -   If there are paper and potentially multiple electronic copies of some information - what happens when it does not agree? What is the source of truth?
+- Sprawl
+    -   Partial adoptions can lead to different pieces of information being stored in different systems which might not be connected making it harder to record and find information which was previous in one place.
+        This can make information recording and retrieval processes more complex, expensive and brittle 
+-   Practical use and interface issues
+    -   Laptops or tablets can introduce interface barriers 
+        -   Lab gloves may hamper the use of touchscreens and track pads
+        -   Free-hand drawing is not always well supported which can be necessary when using notation that is not easily represented in simple text such as mathematical and chemical notation
+        -   Client issues, such as native mobile applications with incomplete featuresets, slow web-apps that may require an always-on connection and not be able to cache local changes
+        -   No spatial memory, whereas in a physical book people may remember where they wrote things and can open it roughly on the right page
+        -   Extra expense, maintenence & procurement issues with hardware suitable for use in your research environment
+
+-   Lack of sufficiently available training on more complex aspects of the ELN solution leading to poor, mis-use, or under-use of ELN features
+
+(rr-rdm-elns-choosing-compare-to-paper)=
+### Always compare to paper notebooks
+
+When picking an ELN remember to always compare the ELN to a paper notebook and not just to other ELNs.
+It is likely paper that you are replacing so this is the benchmark against which to compare.
+Consider the differences between a paper copy of a lab notebook and an electronic copy.
```

Comment:
```suggestion

```
I think this is clear from the first sentence and the sentence after this one.

Reply:
I appreciate that these points may seem redundant but they are all making a slightly different points.

- when you've got your matrix of ELN solutions don't forget to have paper in there too
- why have paper in there: because that's the thing you are replacing so it's the most relevant point of comparison
- don't forget to add the properties of paper notebooks that might be missing from the all the ELN feature lists you've encountered to your comparison matrix
- If one of those features of paper is a deal breaker and is not present in any of the ELNs then you may need to consider other options or delay ELN adoption until you can find one that does have this feature of paper